cdap-client-tests/src/test/java/co/cask/cdap/client/WorkflowClientTestRun.java

```diff
@@ -89,15 +89,10 @@ public void testWorkflowClient() throws Exception {
       NamespaceId.DEFAULT.app(AppWithWorkflow.NAME).workflow(AppWithWorkflow.SampleWorkflow.NAME);
 
     programClient.start(workflowId, false, runtimeArgs);
+    assertProgramRunning(programClient, workflowId);
+    assertProgramRuns(programClient, workflowId, ProgramRunStatus.RUNNING, 1);
     programClient.waitForStatus(workflowId, ProgramStatus.STOPPED, 60, TimeUnit.SECONDS);
```

Comment:
maybe add a `assertProgramRuns` with configurable time, so that no need of calling `assertProgramRunning`?